ui/src/app/pages/app/component.css

```diff
@@ -92,16 +92,12 @@
 }
 
 .background-image {
-  background-image: url('/assets/images/hex-background@2x.png');
-  background-repeat: no-repeat;
-  z-index: -1;
-  background-size: cover;
-  right: 0rem;
-  top: 3.3rem;
+  background-image: url('/assets/images/BG-Pattern.png');
```

Comment:
The new image file has a different naming convention from the old one.

Reply:
That was the filename as given by the designer. Do you think we should rename it to something like `bg-pattern@2x.png`?

Reply:
I don't know what the standard is. I probably wouldn't put an @ sign in the file name though.